react/README.md

```diff
@@ -531,7 +531,7 @@
 
     // good
     render() {
-      return (<div />);
```

Comment:
Reverting it is not a problem, _why I removed it_? I think it would be more consistent throughout all the examples to have it without the parentheses (there is not a single one which contains them) and I would expect that the Parentheses section (https://github.com/airbnb/javascript/tree/master/react#parentheses) is the guide's recommendation for the single line return - which is without parentheses.

Moreover, this particular section the change was made in is only referring to the return statement in render functions, so introducing another variation of a single line return could, under circumstances, lead to unnecessary confusion.

If you want me to change it back, just let me know please.